.github/workflows/build.yml

```diff
@@ -66,14 +67,49 @@ jobs:
             ${{ runner.os }}-gradle-
       - name: Build
         run: ./gradlew build
-      - name: Generate Jacoco report
-        run: ./gradlew jacocoTestReport
+
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'adopt'
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Start the local node
+        run: npx @hashgraph/hedera-local start -d --network local
+
+      - name: Run Integration Tests
```

Comment:
Like the above comment, I think this should be named "Generate Jacoco report" or "Run Tests"

Reply:
Agree. Renamed to 'Run Tests'